src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -276,6 +276,36 @@ internal void AddEntity(IExternalEntity entity, DName displayName = default)
             }
 
             _variables.Add(entity.EntityName, nameInfo);
+        }
+
+        /// <summary>
+        /// Adds a host object schema, that can be referenced in the formula.
+        /// Actual object is added in Runtime config service provider.
+        /// </summary>
+        /// <param name="name">Name of the object.</param>
+        /// <param name="type">Type of the object.</param>
+        /// <param name="getValue">Call back that will retrieve object from the service provider.
+        /// It can throw CustomFunctionErrorException, that fx will convert to an error.</param>

```

Comment:
>CustomFunctionErrorException [](http://example.com/codeflow?start=25&length=28)

nit - you can use <see= > to link to class name within comments. 
This also means if the class gets rename, the comment is updated

Reply:
As discussed offline, we can't do it since the Exception is in the Interpreter.